software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNode.java

```diff
@@ -29,12 +29,10 @@
 import brooklyn.entity.basic.SoftwareProcess;
 import brooklyn.entity.proxying.ImplementedBy;
 import brooklyn.event.AttributeSensor;
-import brooklyn.event.basic.AttributeSensorAndConfigKey;
-import brooklyn.event.basic.PortAttributeSensorAndConfigKey;
-import brooklyn.event.basic.Sensors;
-import brooklyn.event.basic.TemplatedStringAttributeSensorAndConfigKey;
+import brooklyn.event.basic.*;
```

Comment:
No wildcard imports allowed. The standard for Brooklyn Java code is essentially the original Sun **Java Code Conventions** (from 1997) which you can also find at Oracle [here](http://www.oracle.com/technetwork/java/codeconventions-150003.pdf).